### ISSUE TITLE
undefined method `+@' for "":String

### ISSUE THREAD

**Issue by khiav223577** [jnunemaker/httparty#733]
### Reproduction Steps

```rb
HTTParty.get('http://google.com/search', query: { search: 'httparty' }).parsed_response
```

Will get this error:
```
NoMethodError: undefined method `+@' for "":String
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/hash_conversions.rb:31:in `normalize_param'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/hash_conversions.rb:19:in `block in to_params'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/hash_conversions.rb:19:in `each'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/hash_conversions.rb:19:in `map'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/hash_conversions.rb:19:in `to_params'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/request.rb:200:in `normalize_query'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/request.rb:266:in `query_string'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/request.rb:109:in `uri'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/request.rb:215:in `setup_raw_request'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty/request.rb:143:in `perform'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty.rb:613:in `perform_request'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty.rb:527:in `get'
        from /home/khiav/.rvm/gems/ruby-2.2.10/gems/httparty-0.19.0/lib/httparty.rb:646:in `get'
        from (irb):2
        from /home/khiav/.rvm/rubies/ruby-2.2.10/bin/irb:11:in `<main>'
```

**Comment by bethesque** [comment]
Also facing this issue.

**Comment by lacostej** [comment]
The unary + method was added in ruby 2.3.0.

https://docs.ruby-lang.org/en/2.3.0/String.html#method-i-2B-40

The string literal changes introduced in #711 make httparty compatible with ruby 2.3+.

**Comment by jnunemaker** [comment]
@lacostej thanks! I didn't even notice that little change sneak in. Would `"".dup` fix that and work fine with 2.0.0?

**Comment by lacostej** [comment]
> Would "".dup fix that and work fine with 2.0.0?

yes

**Comment by jnunemaker** [comment]
Closed by https://github.com/jnunemaker/httparty/commit/4416141d37fd71bdba4f37589ec265f55aa446ce
